Accordion.js

```diff
@@ -0,0 +1,496 @@
+/** @module deliteful/Accordion */
+define(["dcl/dcl",
+	"decor/sniff",
+	"requirejs-dplugins/Promise!",
+	"delite/register",
+	"delite/KeyNav",
+	"requirejs-dplugins/jquery!attributes/classes",
+	"delite/DisplayContainer",
+	"./ToggleButton",
+	"delite/theme!./Accordion/themes/{{theme}}/Accordion.css"
+], function (dcl, has, Promise, register, KeyNav, $, DisplayContainer, ToggleButton) {
+
+	function setVisibility(node, val) {
+		node.style.display = val ? "" : "none";
+	}
+
+	var animationEndEvent = (function () {
```

Comment:
Ok, but maybe we should put that feature test somewhere it can be used by all the widgets that need it ?